### PR TITLE
Added Manual Scripts

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
Added New files:
	new file:   Test_TC_DGGEN_2_2.yaml
	new file:   Test_TC_DGGEN_2_3.yaml
	new file:   Test_TC_DGGEN_3_1.yaml
	new file:   Test_TC_DGGEN_3_2.yaml
	new file:   Test_TC_MOD_3_4.yaml
	new file:   Test_TC_PCC_3_1.yaml
	new file:   Test_TC_DIAG_TH_NW_2_5.yaml
	new file:   Test_TC_DIAG_TH_NW_3_5.yaml
	new file:   Test_TC_SWTCH_3_2.yaml

  Renamed Scripts
	renamed:    Test_TC_DIAG_TH_NW_1_2.yaml -> Test_TC_DIAG_TH_NW_2_1.yaml
	renamed:    Test_TC_DIAG_TH_NW_1_1.yaml -> Test_TC_DIAG_TH_NW_2_4.yaml
	renamed:    Test_TC_DIAG_TH_NW_2_6.yaml -> Test_TC_DIAG_TH_NW_3_1.yaml
	renamed:    Test_TC_DIAG_TH_NW_2_7.yaml -> Test_TC_DIAG_TH_NW_3_2.yaml
	renamed:    Test_TC_DIAG_TH_NW_2_8.yaml -> Test_TC_DIAG_TH_NW_3_3.yaml
	renamed:    Test_TC_DIAG_TH_NW_2_9.yaml -> Test_TC_DIAG_TH_NW_3_4.yaml
	
	Removed Following Scripts:
	deleted:    Test_TC_EMR_1_1.yaml
	deleted:    Test_TC_GENDIAG_1_2.yaml
	deleted:    Test_TC_BI_1_1.yaml
	deleted:    Test_TC_BI_2_1.yaml
	deleted:    Test_TC_BI_2_2.yaml
	
	Modified Following Scripts:
	modified:   Test_TC_ACT_1_1.yaml
	modified:   Test_TC_IDM_1_1.yaml
	modified:   Test_TC_IDM_1_2.yaml
	modified:   Test_TC_IDM_2_1.yaml
	modified:   Test_TC_IDM_2_2.yaml
	modified:   Test_TC_IDM_3_1.yaml
	modified:   Test_TC_IDM_3_2.yaml
	modified:   Test_TC_IDM_4_1.yaml
	modified:   Test_TC_IDM_4_2.yaml
	modified:   Test_TC_IDM_4_3.yaml
	modified:   Test_TC_IDM_5_1.yaml
	modified:   Test_TC_IDM_5_2.yaml
	modified:   Test_TC_IDM_6_1.yaml
	modified:   Test_TC_IDM_6_2.yaml
	modified:   Test_TC_IDM_6_3.yaml
	modified:   Test_TC_IDM_6_4.yaml
	modified:   Test_TC_IDM_7_1.yaml
	modified:   Test_TC_MOD_1_1.yaml
	modified:   Test_TC_MOD_1_2.yaml
	modified:   Test_TC_MOD_1_3.yaml
	modified:   Test_TC_MOD_2_1.yaml
	modified:   Test_TC_MOD_2_2.yaml
	modified:   Test_TC_MOD_3_1.yaml
	modified:   Test_TC_MOD_3_2.yaml
	modified:   Test_TC_MOD_3_3.yaml
	modified:   Test_TC_SC_4_9.yaml
	modified:   Test_TC_SU_2_8.yaml
	modified:   Test_TC_DGGEN_2_2.yaml
	modified:   Test_TC_DGGEN_2_3.yaml
	modified:   Test_TC_DGGEN_3_1.yaml
	modified:   Test_TC_DGGEN_3_2.yaml
	modified:   Test_TC_MOD_1_2.yaml
	modified:   Test_TC_MOD_1_3.yaml
	modified:   Test_TC_MOD_2_1.yaml
	modified:   Test_TC_MOD_2_2.yaml
	modified:   Test_TC_MOD_3_1.yaml
	modified:   Test_TC_MOD_3_2.yaml
	modified:   Test_TC_MOD_3_3.yaml
	modified:   Test_TC_MOD_3_4.yaml
	modified:   Test_TC_PCC_3_1.yaml
	modified:   Test_TC_SU_2_8.yaml
	modified:   Test_TC_SWTCH_3_2.yaml

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_DIAG_TH_NW_1_1.yaml [hunk 1]
```diff
@@ -11,28 +11,146 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```
[the remark this answers is not in the document]
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_DIAG_TH_NW_1_1.yaml:11]: @jmeg-sfy no still thread network cluster testscripts having this same testcase id ,pls refer [this link](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/thread_nw_diagnostics.adoc)

[the remark this answers is not in the document]
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_DIAG_TH_NW_1_1.yaml:11]: @jmeg-sfy this testscript id is as per latest testplan not sure if there are plans to rename it again ,pls find [link](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/thread_nw_diagnostics.adoc)

**Review by woody-apple — APPROVED** [review]
Fast tracking testing updates

File: src/app/tests/suites/certification/Test_TC_DIAG_TH_NW_1_1.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (raju-apple) [reply src/app/tests/suites/certification/Test_TC_DIAG_TH_NW_1_1.yaml:11]: @jmeg-sfy please file a test plan bug if we need to change it.

**Pull request merged**